### ISSUE TITLE
test_all_examples.py 

### ISSUE THREAD

**Issue by TonePoems** [COLAB2/midca#10]
The file midca/tests/test_all_examples.py throws the following errors on files which should be more specific to the error being related to Running ROS on a machine not connected to Neo.
examples/ObjectDetector.py...............................................................[FAILED during initialization]
examples/Calibrate.py............................................................................[FAILED during initialization]

**Comment by sampathgogineni** [comment]
Yes, that is true there are example files in there that need other third-party applications like ROS to test. So, I have updated the test_all_examples.py file to ignore those files for now.

**Comment by mcox25** [comment]
Add global var to indicate libraries and packages that are active (e.g., ROS).

**Comment by mcox25** [comment]
So why is test_all_examples.py in a separate directory called "tests" ?
